extensions/wikia/AdEngine/js/lookup/adapter/matomy.js

```diff
@@ -0,0 +1,88 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.adapter.matomy', [
+	'ext.wikia.adEngine.adContext'
+], function (adContext) {
+	'use strict';
+
+	var bidderName = 'appnexus',
+		placementId = '7071682',
+		slots = {
+			oasis: [
+				'TOP_RIGHT_BOXAD',
+				'LEFT_SKYSCRAPER_2',
+				'LEFT_SKYSCRAPER_3',
+				'INCONTENT_BOXAD_1',
+				'PREFOOTER_LEFT_BOXAD',
+				'PREFOOTER_RIGHT_BOXAD'
+			],
+			mercury: [
+				'MOBILE_TOP_LEADERBOARD',
+				'MOBILE_IN_CONTENT',
+				'MOBILE_PREFOOTER'
+			]
+		};
+
+	function isEnabled() {
+		// TODO: enable with wgCountries
+		return true;
+	}
+
+	function prepareAdUnit(slotName) {
+		return {
+			code: slotName,
+			sizes: [
+				[300, 250]
+			],
+			bids: [
+				{
+					bidder: bidderName,
+					params: {
+						placementId: placementId
+					}
+				}
+			]
+		};
+	}
+
+	function configureOasisSlots() {
+		var context = adContext.getContext(),
+			i,
+			slotName;
+
+		if (context.targeting.pageType === 'home') {
```

Comment:
It's the same as https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js#L129.
Any ideas how we can generalise it?